File: ProgramasPython/URI.Problems/problem1060.py
## 
## Numeros Positivos
## URI ONLINE - Problema 1060 - Iniciante
##
## Created by Jon on 16.03.2016
##	

# -*- coding: utf-8 -*-

cont = 0

for x in range(1,7):
	valor = input()
	if valor>0:
		cont+=1
		
print ("%d valores positivos"%(cont))

File: Programas_C/URI/problem1060.c
#include <stdio.h>

int main(){
	int cont = 0, i = 6;
	double valor;
	
	while(i--){
		scanf("%lf", &valor);
		if (valor > 0){
			cont++;
		}
	} 

	printf("%i valores positivos\n", cont);

	return 0;
}